src/ngraph/frontend/onnx_import/wrappers/node.cpp

```diff
@@ -15,8 +15,8 @@
 *******************************************************************************/
 
 #include "node.hpp"
-#include "graph.hpp"
-#include "ops_bridge.hpp"
```

Comment:
This is fundamentally incorrect!!!! Why `wrapper` needs access to `ops_bridge` ???